docs/configfile.adoc

```diff
@@ -60,6 +61,9 @@ a| To run Smart Testing with `SCM` configuration. You can either define `range`
 
 a| autocorrect
 a| This configures Smart Testing to auto correct misspelled strategies to the closest one. For example in case of user set strategies to `nwe`, if `autocorrect` is enabled then it is automatically changed to `new`.
+
+a| customStrategies
+a| A list of custom strategies in the form of key/value. It is important to notice that the _key_ part must be prefixed with `const:mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/StrategyDependencyResolver.java[name="SMART_TESTING_STRATEGY_PREFIX"]`.
```

Comment:
Constant `SMART_TESTING_STRATEGY_PREFIX` is showing `.` as a result in final view. Can you generate documentation locally & check it?

Reply:
I did it in this way so user see that he still needs to put the dot, so he don't do (I know that it is stupid) but they don't do `smart.testing.strategy;cool=org.....` notice that I am using `;` as separator

Reply:
![screenshot from 2017-10-17 16-29-49](https://user-images.githubusercontent.com/7941617/31661469-74ec1954-b358-11e7-92bd-f4ab025a029e.png)

Reply:
Look how it's displaying in resulting documents